applications/GeoMechanicsApplication/tests/dsettlement/consolidation_uniform_load/coarse_mesh/ProjectParameters_stage2.json

```
@@ -0,0 +1,255 @@
+{
+  "problem_data": {
+    "problem_name"                    : "mesh",
+    "start_time"                      : 0,
+    "end_time"                        : 86.4,
+    "echo_level"                      : 1,
+    "parallel_type"                   : "OpenMP",
+    "number_of_threads"               : 1
+  },
+  "solver_settings": {
+    "solver_type"                     : "U_Pw",
+    "model_part_name"                 : "PorousDomain",
+    "domain_size"                     : 2,
+    "model_import_settings": {
+      "input_type"                    : "mdpa",
+      "input_filename"                : "mesh"
```

Here we can use the following input type to ensure the mdpa isn't read again. This also holds for stage 3 and stage 2/3 of the fine mesh.
```suggestion
      "input_type"                    : "use_input_model_part"
```